internal/handlers/common/projection.go

```diff
@@ -200,6 +200,9 @@ func ValidateProjection(projection *types.Document) (*types.Document, bool, erro
 		}
 
 		if *inclusion != inclusionField {
+			if key == "_id" {
+				continue
+			}
```

Comment:
Could you add an integration test for it please :pray:? We could add it in this file where we test query projection https://github.com/FerretDB/FerretDB/blob/main/integration/query_projection_test.go :hugs:

Reply:
sure

Reply:
done